src/components/AssignmentTexter/ContactController.jsx

```diff
@@ -187,9 +187,11 @@ export class ContactController extends React.Component {
       // console.log('getContactData length', newIndex, getIds.length)
       this.setState({ loading: true });
       const contactData = await this.props.loadContacts(getIds);
-      const {
-        data: { getAssignmentContacts }
-      } = contactData;
+      let getAssignmentContacts;
+      if (contactData && contactData.data) {
+        getAssignmentContacts = contactData.data.getAssignmentContacts;
```

Comment:
This seems unrelated to most of the other changes, what's this in here for? Is it related to a bug?

Reply:
This change is from the PR that fixed the GitHub Actions failing automated tests: https://github.com/MoveOnOrg/Spoke/pull/2090

I cherry-picked the changes from that PR into this PR so that the automated tests wouldn't all fail.